SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -137,5 +175,22 @@ private static Configuration OpenConfiguration()
             var executionFileMap = new ExeConfigurationFileMap { ExeConfigFilename = FilePath };
             return ConfigurationManager.OpenMappedExeConfiguration(executionFileMap, ConfigurationUserLevel.None);
         }
+
+        private void RestoreConfigurationFile()
+        {
+            try
+            {
+                if (File.Exists(FilePath))
+                {
+                    File.Delete(FilePath);
+                    configuration = OpenConfiguration();
+                }
+                AppCenterLog.Info(AppCenterLog.LogTag, "Configuration file is successfully restored from backup.");
```

Comment:
There is no backup anymore, this log should be updated.
Also the log should be inside the if block.